private/outdated.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [ -f "private/tools/prebuilt/outdated_deploy.jar" ]; then
```

Comment:
I'm not sure if there is a better way to check if the action is running inside the rules_jvm_external repo or outside of it.

Reply:
As far as I can tell https://github.com/bazelbuild/rules_jvm_external/issues/445 is still an open issue because JAVABASE is not exposed to repository rules.  If we fix JAVABASE for the java that coursier uses we should fix it here also but I don't currently see a straightforward fix otherwise.